### ISSUE TITLE
How to press teensy reset button?

### ISSUE THREAD

**Issue by anildigital** [ErgoDox-EZ/docs#8]
Should I put pin in the hole?

**Comment by ezuk** [comment]
Exactly! Don't bend it or anything -- one person broke a pin off by mistake
and it's still inside his keyboard. :)

**Comment by rvl** [comment]
I started with `wally-cli` and it told me to "Press the reset button of your keyboard." I had a suspicion that it was the little hole, then went searching for a paperclip. It might be worth adding a note to the [quickstart doc](https://ergodox-ez.com/pages/getting-started) that the reset button is in there, and you use a paperclip, and that it puts the QMK firmware into flashing mode (keys stop working). It would also be helpful to include a paperclip with the keyboard.

**Comment by yati-sagade** [comment]
how does one press this key? Mine simply does not want to work. I am afraid I might even have gone too far. Pretty dissapointing that a simple tool to do this was not included.

**Comment by rvl** [comment]
@yati-sagade 

1. Find the hole for the reset button. On the Ergodox-EZ it's in the upper-right of the keyboard. On the Planck-EZ it's on the underside of the keyboard, towards the upper-left.
2. Insert a straightened paperclip into the hole - straight down. I think it's better to use a paperclip than a pin because the end isn't sharp.
3. Gently press down and you should hear a little click from the button engaging.

**Comment by joshnoe** [comment]
This hole is way too small. Most paperclips won't fit so you'll likely need to go out and buy a needle or something similar. Why they made the hole this tiny and then didn't include a special tool is beyond me.

**Comment by tacerus** [comment]
A SIM eject tool from a Samsung phone works fine for me. The SIM eject tool shipped with Apple phones is too short.
